javascript-loops/main.js

```diff
@@ -0,0 +1,68 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+
+function getNumbersToTen() {
+  var number = [];
+  var currentNumber = 1;
+  while (currentNumber < 10) {
+    number.push(currentNumber);
+    currentNumber++;
+  }
+  return number;
+}
```

Comment:
`getNumbersToTen` should include `10`:

---

![image](https://user-images.githubusercontent.com/16143771/152256855-0d665181-fbb7-4150-9ff3-580e8244516d.png)

---

![image](https://user-images.githubusercontent.com/16143771/152256883-a5b0be73-299b-48d5-a14b-24726ea3befe.png)